### ISSUE TITLE
Cannot find module _C.pyd

### ISSUE THREAD

**Issue by Helyan47** [TimDettmers/bitsandbytes#319]
Hi, when I'm using the Dreamboth LoRA, the command line shows the next error:

enable LoRA for U-Net
prepare optimizer, data loader etc.
use AdamW optimizer | {}
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 2000
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 1000
  num epochs / epoch数: 1
  batch size per device / バッチサイズ: 2
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 1000
steps:   0%|                                                                                  | 0/1000 [00:00<?, ?it/s]epoch 1/1
Could not find module 'E:\IA\Plugins\Kohya\kohya_ss\venv\Lib\site-packages\xformers\_C.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
WARNING:root:WARNING: Could not find module 'E:\IA\Plugins\Kohya\kohya_ss\venv\Lib\site-packages\xformers\_C.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
Need to compile C++ extensions to get sparse attention suport. Please run python setup.py build develop

The file exists in the specified location. Anyone has any idea what the problem is?

Before this error I had other related with libbitsandbytes_cuda116.dll. I solved it like it specifies in this issue: [https://github.com/TimDettmers/bitsandbytes/issues/161](url)
Thanks you
